.github/workflows/release.yml

```
@@ -0,0 +1,17 @@
+name: CD Pipeline
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build and publish to pypi
+        uses: JRubics/poetry-publish@v1.10
+        with:
```

I think we can [leave out the dev requirements](https://github.com/marketplace/actions/publish-python-poetry-package#ignore_dev_requirements) (e.g. [`tox`, `pytest` and `python-dotenv`](https://github.com/phillymedia/vestapol/blob/b1086f0dac87f5319d869597e58ef0da9c45d4be/pyproject.toml#L18-L21)) in our published package.
```suggestion
        with:
          ignore_dev_requirements: "yes"
```